uaclient/config.py

```diff
@@ -550,10 +556,13 @@ def _attached_status(self) -> "Dict[str, Any]":
         from uaclient.entitlements import ENTITLEMENT_CLASSES
 
         response = copy.deepcopy(DEFAULT_STATUS)
-        contractInfo = self.machine_token["machineTokenInfo"]["contractInfo"]
+        machineTokenInfo = self.machine_token["machineTokenInfo"]
```

Comment:
Should we redact that info on the output if a non-root user runs that command ?

Reply:
We're only grabbing the `machineId` which I think is okay for non-root users? We decided not to include the `machineToken` because it is a secret.

Reply:
Fair enough, the `machineToken` was the one I was actually thinking about